clusters/build-clusters/01_cluster/openshift-ingress-operator/default_ingresscontroller.yaml

```diff
@@ -0,0 +1,15 @@
+#We put only the customized fields in git
```

Comment:
Can we put this into the "common without api ci folder" as this is not specfic to any one cluster?

Reply:
Add more fields we touched.
It is specific to build01 (my bad naming strategy slows us down).

`build02` has not yet set up infra? Do we want to do it all together?
More over, do we want to do this to vsphere as well?

Reply:
We want this for build02, yes. Vsphere is a weird one as we do not own it

Reply:
If that is not for `vsphere`, then we cannot put it into https://github.com/openshift/release/tree/master/clusters/build-clusters/common_except_app.ci

The one for build02 is another PR.